slf4j-android/pom.xml

```diff
@@ -20,7 +21,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>${project.version}</version>
```

Comment:
Can you revert the POM changes? I don't think you meant to include them in the PR.